addons/l10n_uy/demo/account_demo.py

```diff
@@ -1,20 +1,613 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import api, models
+import logging
+from odoo import api, models, Command
+import time
+
+from odoo.exceptions import UserError, ValidationError
+_logger = logging.getLogger(__name__)
 
 
 class AccountChartTemplate(models.AbstractModel):
 
     _inherit = "account.chart.template"
 
+    @api.model
+    def _get_demo_data(self, company=False):
+        if company.account_fiscal_country_id.code == "UY":
+            return {
+                'account.move':
+                    self._get_demo_data_move(company),
+                'account.move.reversal':
+                    self._get_demo_data_move_reversal(company),
+                'res.partner':
+                    self._get_demo_data_res_partner(company),
+            }
+        else:
+            return super(AccountChartTemplate)._get_demo_data(self,
+                                                              company=False)
```

Comment:
You seem to have a linter that makes you add a line when your line is too long, but we tend to have a soft limit at 120 char, which your file never attains.
Could you reput this on a single line?